brightener.h

```diff
@@ -1,11 +1,44 @@
 #include <cstdint>
 #include "image.h"
 
-class ImageBrightener {
-    private:
-        std::shared_ptr<Image> m_inputImage;
-    public:
-        ImageBrightener(std::shared_ptr<Image> inputImage);
-        int BrightenWholeImage();
-        bool AddBrighteningImage(std::shared_ptr<Image> imageToAdd, int& attenuatedCount);
-};
+namespace ImageProcessor
+{
+
+    void ExecutePixelRunner(std::shared_ptr<Image> inputImage, const std::shared_ptr<Image>& imageToAdd, int& attenuatedCount)
+    {
+        auto BrightnerFun = [&attenuatedCount](uint8_t pixelGrayscale, uint8_t pixelGrayscaleToAdd) {
```

Comment:
Related to my comment in `image.h` - Why not have multiple brightening functions, one to add a fixed number to each pixel, another to add respective pixels of `imageToAdd`, etc?

Reply:
Done